app/assets/javascripts/discourse/tests/setup-tests.js

```diff
@@ -389,19 +380,8 @@ function setupTestsCommon(application, container, config) {
     return true;
   };
 
-  if (isLegacyEmber()) {
-    Object.keys(requirejs.entries).forEach(function (entry) {
-      if (shouldLoadModule(entry)) {
-        require(entry, null, null, true);
-      }
-    });
-  } else {
-    // Ember CLI
-    const emberCliTestLoader = require("ember-cli-test-loader/test-support/index");
-    emberCliTestLoader.addModuleExcludeMatcher(
-      (name) => !shouldLoadModule(name)
-    );
-  }
+  const emberCliTestLoader = require("ember-cli-test-loader/test-support/index");
```

Comment:
I think this can now be moved to the top as a regular import? `import TestLoader from "ember-cli-test-loader/test-support";`

Reply:
Updated